ipywidgets/widgets/widget_float.py

```diff
@@ -64,74 +64,79 @@ def _validate_max(self, proposal):
+    color : str Unicode color code (eg. '#C13535')
+        color of the value displayed
     """
     _view_name = Unicode('FloatTextView').tag(sync=True)
-    _model_name = Unicode('IntTextModel').tag(sync=True)
+    _model_name = Unicode('FloatTextModel').tag(sync=True)
 
 
 @register('Jupyter.BoundedFloatText')
 class BoundedFloatText(_BoundedFloat):
     """ Displays a float value within a textbox. Value must be within the range specified.
-	For a textbox in which the value doesn't need to be within a specific range, use FloatText.
-
-	Parameters
-	----------
-	value : float
-	    value displayed
-	min : float
-	    minimal value of the range of possible values displayed
-	max : float
-	    maximal value of the range of possible values displayed
-	description : str
-	    description displayed next to the textbox
-	color : str Unicode color code (eg. '#C13535'), optional
-	    color of the value displayed
+
+    For a textbox in which the value doesn't need to be within a specific range, use FloatText.
+
+    Parameters
+    ----------
+    value : float
+        value displayed
+    min : float
+        minimal value of the range of possible values displayed
+    max : float
+        maximal value of the range of possible values displayed
+    description : str
+        description displayed next to the textbox
+    color : str Unicode color code (eg. '#C13535')
+        color of the value displayed
     """
     _view_name = Unicode('FloatTextView').tag(sync=True)
-    _model_name = Unicode('IntTextModel').tag(sync=True)
+    _model_name = Unicode('FloatTextModel').tag(sync=True)
 
 
 @register('Jupyter.FloatSlider')
 class FloatSlider(_BoundedFloat):
     """ Slider/trackbar of floating values with the specified range.
 
-	Parameters
-	----------
-	value : float
-	    position of the slider
-	min : float
-	    minimal position of the slider
-	max : float
-	    maximal position of the slider
-	step : float
-	    step of the trackbar
-	description : str
-	    name of the slider
-	orientation : {'vertical', 'horizontal}, optional
-        default is horizontal
-	readout : {True, False}, optional
-	    default is True, display the current value of the slider next to it
-	slider_color : str Unicode color code (eg. '#C13535'), optional
-	    color of the slider
-	color : str Unicode color code (eg. '#C13535'), optional
-	    color of the value displayed (if readout == True)
+    Parameters
+    ----------
+    value : float
+        position of the slider
+    min : float
+        minimal position of the slider
+    max : float
+        maximal position of the slider
+    step : float
+        step of the trackbar
+    description : str
+        name of the slider
+    orientation : {'horizontal', 'vertical'}
+        default is 'horizontal', orientation of the slider
+    readout : {True, False}
+        default is True, display the current value of the slider next to it
+    readout_format : std
```

Comment:
I did not put a link to the language spec because I want to be sure that it is not a strict superset of the Python 3.1's built-in format specification mini-language https://docs.python.org/release/3.1.3/library/string.html#formatspec.